testsuite/MDAnalysisTests/topology/test_fhiaims.py

```diff
@@ -40,15 +39,7 @@ def test_names(self, top):
         assert_equal(top.names.values,
                      ['O', 'H', 'H', 'O', 'H', 'H'])
 
-    def test_types(self, top):
```

Comment:
Are these tests being moved somewhere else? (I can't see them anywhere) - they are important in ensuring that we retain the same behaviour.

Reply:
It would be good to rework that test so it does check that the types are the expected ones in a universe created with the default behaviour. This would make sure we do not break the code of people relying on these types and this parser.

Reply:
good idea, I can add it to all parsers not just this one